### DISCUSSION TITLE
Is this project still maintained ?

### DISCUSSION THREAD

**Discussion by fidgi** [loopbackio/loopback-next#7492] [Q&A] [answered]
Hi,

Since the a couple of months , the number of bug fixes and evolutions that are actually released is decreasing.
Only renovate-bot seems to be active. Are the current owners mentionned in the README still involved ? 
Does the community have to worry about the future of this project ?

Regards,
PS: thanks to the team for the great job done so far

**Comment by achrinza** [comment, marked as answer]
Hi @fidgi, thanks for initiating a discussion!

To answer: **Yes, this project is still maintained.**

Several maintainers (myself included) are utilizing LoopBack 4 in production, and for upcoming projects within our own organization(s). Hence, actively maintaining LoopBack 4 is still a priority for many of us.

LoopBack 4 is effectively a community-driven project; This means that it relies on the community and maintainers to work on it on their spare time, most without direct monetary support. Hence why we strongly recommend the community to contribute their own pull requests for bug fixes and feature additions.

That's not to say that the maintainers won't fix bugs or bring in new features; We are more than happy to continue supporting the community and the project within the constraints of our spare time. It's just that, for differing reasons, we may have priorities elsewhere that may prevent us from contributing as much as we wish to. For example, I'm currently working on new ORM integrations with LoopBack 4, and I believe that many of the other maintainers have similar integrations that they're working on to make LoopBack 4 fit their projects' needs.

Unlike open source projects which are driven by a single company, LoopBack 4 has a more distributed contribution model; The maintainers listed on the README are from different organizations, have the rights to review and merge pull requests, and most have a say in the technical direction of the project. Hence, the project isn't going to be "let go" all of a sudden due to a shift in an single organization's interests.

I don't think the community has to worry about the future of the project; It's a community-driven effort and we are more than happy to help the community to merge their contributions into the project.

Hope this clarifies the main points of your question :)

**Reply by fidgi** [reply]
@achrinza , thanks for the quick answer. 
it makes me more confident about the future of the project.

And I'll try to contribute when I'll be more confident in my Typescript/Javascript skills.

regards,
